DEPRECATION.md

```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
 [link-enable-https]:
   https://cube.dev/docs/deployment/production-checklist#enable-https
 
-### `hearBeatInterval`
-
-**Deprecated in Release: v0.23.8**
-
-This option for [`@cubejs-client/ws-transport`][link-hearbeatinterval] has been
-replaced by `heartBeatInterval`.
-
-[link-hearbeatinterval]:
-  https://cube.dev/docs/@cubejs-client-ws-transport#web-socket-transport-hear-beat-interval
+### Embedding Cube.js within Express
 
-### Node.js 8
+**Deprecated in Release: v0.24.0**
 
-**Deprecated in Release: v0.22.4**
+Embedding Cube.js into Express applications is deprecated due to performance and
+reliability considerations. [Read more about this change
+here][link-cube-docker].
 
-Node.js 8 reached [End of Life on December 31, 2019][link-nodejs-eol]. This
-means no more updates. Please upgrade to Node.js 10 or higher.
+Developers are encouraged to [migrate to the new `cube.js` configuration
+file][link-migration] and deploy Cube.js as a microservice (or multiple
+microservices, if necessary).
 
-[link-nodejs-eol]: https://github.com/nodejs/Release#end-of-life-releases
+[link-cube-docker]: https://cube.dev/blog/cubejs-loves-docker
+[link-migration]:
+  https://cube.dev/docs/configuration/overview#migrating-from-express-to-docker
 
-### Absolute import for @cubejs-backend/query-orchestrator
+### Absolute import for `@cubejs-backend/query-orchestrator`
 
 **Deprecated in Release: v0.24.2**
 
-Reason: Absolute imports highly depend on a path, and all API becomes public. Now we started to provide public API as `export` from the module.
+Absolute imports highly depend on a path, and all API becomes public. We now
```

Comment:
How about "Absolute imports are highly dependent on a path, "?

Reply:
Not sure, I think it's better to keep the API part but perhaps reword it to the following:

```suggestion
Absolute imports highly depend on a path, and their API becomes public. We now
```